en/physics/physics-raycast.md

```diff
@@ -1,92 +1,117 @@
 # Raycast Detection
 
-__Raycast detection__ is a very important function and is often used to judge various situations. The essence is to make a **intersection judgment** between a ray and another shape, as shown in the figure below.
+Raycast detection is a **intersection determination** of one ray and another shape, as shown in the figure below.
```

Comment:
"Raycast detection is a **intersection determination** of one ray and another shape" seems weired.